dotcom-rendering/src/components/HeadlineExample.stories.tsx

```diff
@@ -0,0 +1,61 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import type { Decorator, Meta, StoryObj } from '@storybook/react';
+import { paletteDeclarations } from '../palette';
+import { HeadlineExample } from './HeadlineExample';
+
+// ----- Meta ----- //
+
+const meta: Meta<typeof HeadlineExample> = {
+	title: 'components/HeadlineExample',
+	component: HeadlineExample,
+};
+
+export default meta;
+
+// ----- Decorators ----- //
+
+/**
+ * Creates storybook decorator used to wrap components in an element
+ * containing the light or dark mode palette colours.
+ *
+ * @param colourScheme Choose whether to use the light or darPerformanceNavigation.type palette.
```

Comment:
I love `PerformanceNavigation.type` as much as the next person, but not sure what it’s doing there…